server/zanata-war/src/main/webapp/WEB-INF/layout/language/settings-tab.xhtml

```diff
@@ -104,8 +104,12 @@
         </div>
 
         <div class="l--push-top-1">
-          <a4j:commandLink action="#{languageAction.saveSettings()}" styleClass="loader button button--primary"
-            render="language-info" onbegin="jQuery(this).attr('disabled', 'disabled')" oncomplete="jQuery(this).removeAttr('disabled').removeClass('is-active')">
+          <a4j:commandLink id="save-language-settings"
+                           action="#{languageAction.saveSettings()}"
+                           styleClass="loader button button--primary"
+                           render="language-info"
+                           onbegin="jQuery(this).attr('disabled', 'disabled')"
+                           oncomplete="jQuery(this).removeAttr('disabled').removeClass('is-active')">
```

Comment:
I'm glad you kept the order the same!

But this indentation isn't standard. I think we indent continuations by 2 spaces in XML.

Reply:
Oops, fixed